src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -82,22 +82,30 @@ def iot_hub_sku_list(client, hub_name, resource_group_name=None):
     return client.get_valid_skus(resource_group_name, hub_name)
 
 
-def iot_hub_consumer_group_create(client, hub_name, consumer_group_name, resource_group_name=None, event_hub_name='events'):
+def iot_hub_consumer_group_create(client, hub_name, consumer_group_name, resource_group_name=None, event_hub_name=None):
+    if event_hub_name is None:
```

Comment:
I just tested giving an IDs component a default value and it worked fine whether I was using the default or a non-default value as the name or in the ID. Can you be give a code snippet that shows which aliases (the register_cli_argument entries) are contributing to the ID parameter for this command?